tests/ui/unsafe-fields/auto-traits.rs

```diff
@@ -0,0 +1,23 @@
+//@ compile-flags: --crate-type=lib
+
+#![feature(auto_traits)]
+#![feature(unsafe_fields)]
+#![allow(dead_code, incomplete_features, unconditional_recursion)]
```

Comment:
Please remove the `unconditional_recursion`. No reason not to just split this out into a `fn main() {}` or something that just calls both of the functions.

Reply:
No need for `dead_code`, that's enabled in the UI test suite unconditionally.

Reply:
[Done](https://github.com/rust-lang/rust/compare/baec50ee0b8afc7433c3bd2f2b862a21addd2f57..9d604ce5704f06017fa06a1ebacd496f9cebb4eb#diff-f20103ee4791c97758b52ea3c6d14d523b123dc658f33307bfe72217089d48f8L5-R7)